app/models/abstract-node.ts

```diff
@@ -15,6 +15,9 @@ export default class AbstractNodeModel extends BaseFileItem {
 
     @attr('array') currentUserPermissions!: Permission[];
 
+    @hasMany('review-action', { inverse: 'target' })
```

Comment:
Now that we add this relationship on the parent model, should we delete it from the child classes?